Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Set the base image to vivek77/jenkinsbot
+FROM vivek77/jenkinsbot
```

Comment:
Let's add a http link to the base image.

We'd better place the Dockerfile(s) in the same git repo.